vue-components/src/components/Button.vue

```diff
@@ -13,12 +21,12 @@ Vue.use( VueCompositionAPI );
 /* eslint-disable no-trailing-spaces */
 /**
  *  An interactive element signaling a single-step action that will occur when the user clicks or taps on it.
- * 
+ *
  * Known issues:
- * 
- * * The styles defined on `:focus` do not apply in Safari and Firefox on macOS.  
- * This seems to be a desired behavior and not a bug  
- * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus  
```

Comment:
This annoys me every time I see it, but these trailing double-spaces are actually meaningful: they add newlines to the documentation. I think I found an alternative to that at some point but forgot to change it. Either way, this shouldn't happen in this PR :/

Reply:
ugh, they are auto-removed by my IDE and I didn't notice it this time. Will fix

Reply:
fixed!